README.md

```diff
@@ -23,3 +23,7 @@
 ```bash
 npx speed-cloudflare-cli
 ```
+or
+```bash
+docker run --rm ghcr.io/luc122c/speed-cloudflare-cli:master
```

Comment:
if/when ready to merge, this probably needs to be renamed for the target repo this links to?

Reply:
The workflow file pushes to `github.repository` so currently, it publishes to my fork. You're correct that when merging, it would need updating to the new location.